test/e2e/autoscale.go

```diff
@@ -158,7 +158,7 @@ func generateTraffic(
 			totalRequests++
 			if res.Code != http.StatusOK {
 				ctx.t.Logf("Status = %d, want: 200", res.Code)
-				ctx.t.Logf("URL: %s Duration: %v Body:\n%s", res.URL, res.Latency, string(res.Body))
```

Comment:
did you mean to leave this in here, doesn't seem related? or did the dependency change cause us to lose URL as a field from res?

Reply:
Yeah the dep did change ever so slightly

Reply:
what a pain, wonder if it's worth keeping the replace on `vegeta` so we have the URL field, seems like potentially quite a useful bit of debug to have? I did a quick try of just bumping the dep in the regular require section out of interest but looks like go-licensechecker gets confused because there's a transitive dependency (github.com/josharian/intern) whose license.md file is lowercase 😢.

Reply:
unless the URL is actually static here anywhere because we pass a single target to StaticTargeter 🤔?

Reply:
ack, I can actually work on bumping the dep first, but it requires a pkg bump. I'll review where the pkg things are used and then submit a few PRs.

Reply:
Done!